fix: Wrong color in widget see new message button

iamsivin opened the pull request:
# Pull Request Template

## Description

Fixes https://github.com/chatwoot/chatwoot/issues/5977

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## How Has This Been Tested?

**Screenshot**

<img width="225" alt="image" src="https://user-images.githubusercontent.com/64252451/206121588-5703a1e2-4176-4c6a-938c-84f6836772e4.png">



## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

pranavrajs commented on app/javascript/widget/components/UnreadMessageList.vue:
```diff
@@ -27,7 +27,12 @@
+        <span
+          class="flex items-center"
+          :style="{
+            color: widgetColor,
```
One problem I can see is that if the user selected white, the button would become invisible.

iamsivin replied:
@pranavrajs Is is fine? 
<img width="225" alt="image" src="https://user-images.githubusercontent.com/64252451/206360160-9492448a-2a1d-436e-9d7e-1e8adb813b17.png">

https://github.com/chatwoot/chatwoot/pull/6033/commits/3e06cb6d928fdf5b1d2a81fabbb50efa2792ff60

iamsivin commented:
@pranavrajs @nithindavid Made some changes in the SDK file and will change the colour dynamically in the lighter background colours. https://github.com/chatwoot/chatwoot/pull/6033/commits/abde10607f0e5308ac01e794b2e250b9af81419b

https://user-images.githubusercontent.com/64252451/206650920-026eb6c2-5760-46d8-9cfc-debfd1ee7fe7.mov

iamsivin commented:
@pranavrajs, Can you take a look?
1. Updated bubble icon from `img` to `svg`.
2. The text and bubble colour in the widget bubble and the `show new message` button colour will update dynamically as per the brightness of the background. If its lighter, it will change to the default colour `#37546D`(s-700)

@nithindavid, I Updated the PR with the changes you suggested in the call. Can you take a look?

**Preview**

https://user-images.githubusercontent.com/64252451/206752916-4d7a835b-cc24-4d4c-a79f-f8eb905442e9.mov

nithindavid commented on app/javascript/sdk/sdk.js:
```diff
@@ -107,7 +107,11 @@ export const SDK_CSS = `
 
-.woot-widget-bubble.woot-widget--expanded img {
+.woot-widget-bubble.woot-widget--expanded.woot-widget-bubble-color--lighter div{
+  color: 	#37546D;
```
Please move this as a variable since we are using this in 3 places

iamsivin replied:
Done

nithindavid reviewed (approved):
Tested in local and working as expected.
